packages/agoric-cli/src/commands/inter.js

```diff
@@ -0,0 +1,376 @@
+  const pprint = v => JSON.stringify(v, bigintReplacer, 2);
+
+  const liquidationCmd = interCmd
+    .command('liquidation')
+    .description('liquidation commands');
+  liquidationCmd
+    .command('status')
+    .description(
+      `show amount liquidating, oracle price
+
+For example:
+
+{
+  "liquidatingCollateral": "10IbcATOM",
+  "liquidatingDebt": "120IST",
+  "price": "12.00 IST/IbcATOM"
+}
+`,
+    )
+    .option('--manager [number]', 'Vault Manager', Number, 0)
+    .action(async opts => {
+      const { agoricNames, vstorage, fromBoard } = await rpcTools();
+      const unBoard = txt =>
+        storageHelper.unserializeTxt(txt, fromBoard).at(-1);
+      const [metrics, quote] = await Promise.all([
+        vstorage
+          .readLatest(`published.vaultFactory.manager${opts.manager}.metrics`)
+          .then(unBoard),
+        vstorage
+          .readLatest(`published.vaultFactory.manager${opts.manager}.quotes`)
+          .then(unBoard),
+      ]);
+      stdout.write(
+        pprint(fmtMetrics(metrics, quote, values(agoricNames.vbankAsset))),
+      );
+      stdout.write('\n');
+    });
+
+  const bidCmd = interCmd
+    .command('bid')
+    .description('auction bidding commands');
+
+  bidCmd
+    .command('by-price')
+    .description('Print an offer to bid collateral by price.')
+    .requiredOption('--price [number]', 'bid price', Number)
+    .requiredOption('--giveCurrency [number]', 'Currency to give', Number)
+    .requiredOption('--wantCollateral [number]', 'bid price', Number)
+    .option('--collateralBrand [string]', 'Collateral brand key', 'IbcATOM')
+    .option('--offerId [number]', 'Offer id', String, `bid-${clock()}`)
+    .action(
+      /**
+       * @param {{
+       *   price: number,
+       *   giveCurrency: number, wantCollateral: number,
+       *   collateralBrand: string,
+       *   offerId: string,
+       * }} opts
+       */
+      async ({ collateralBrand, ...opts }) => {
+        const { agoricNames } = await rpcTools();
+        const offer = Offers.auction.Bid(agoricNames.brand, {
+          collateralBrandKey: collateralBrand,
+          ...opts,
+        });
+        outputExecuteOfferAction(offer, stdout);
+        stderr.write(
+          'Now use `agoric wallet send ...` to sign and broadcast the offer.\n',
+        );
+      },
+    );
+
+  const parsePercent = v => {
+    const p = Number(v);
+    if (!(p >= -100 && p <= 100)) {
+      throw new InvalidArgumentError('must be between -100 and 100');
+    }
+    return p / 100;
+  };
+
+  bidCmd
+    .command('by-discount')
+    .description(
+      `Print an offer to bid on collateral based on discount from oracle price.`,
+    )
+    .requiredOption(
+      '--discount [percent]',
+      'bid discount (0 to 100) or markup (0 to -100) %',
+      parsePercent,
+    )
+    .requiredOption('--giveCurrency [number]', 'Currency to give', Number)
+    .requiredOption('--wantCollateral [number]', 'bid price', Number)
```

Comment:
price is a ratio. maybe…
```suggestion
    .requiredOption('--wantCollateral [number]', 'collateral expected for the currency', Number)
```